app/components/transaction.js

```diff
@@ -16,12 +18,23 @@ function NewComponent () {
 
 NewComponent.prototype.render = function () {
   const props = this.props
-  const { transaction } = props
+  const { transaction, isMobile } = props
+
+  console.log(transaction)
+
+  if(isMobile) {
+    return TransactionMobile(transaction)
```

Comment:
Nit: it looks like `TransactionMobile` is a render function rather than its own component, so conventionally it would start with a lower-cased letter. The capital implies that it's a class/component.